pkg/analysis/namingconventions/analyzer.go

```diff
@@ -36,9 +36,11 @@ type analyzer struct {
 	conventions []Convention
 }
 
-// newAnalyzer creates a new analysis.Analyzer for the namingconventions
+// NewAnalyzer creates a new analysis.Analyzer for the namingconventions
 // linter based on the provided config.
-func newAnalyzer(cfg *Config) *analysis.Analyzer {
+// This function is exported to allow other linters to wrap namingconventions
+// with fixed configuration.
+func NewAnalyzer(cfg *Config) *analysis.Analyzer {
```

Comment:
Are we certain this needs to be changed to exported? I thought we avoided that on the other PR